packages/cf.js/src/app-instance.ts

```diff
@@ -9,8 +9,16 @@ import {
   Node
 } from "@counterfactual/common-types";
 import { BigNumber } from "ethers/utils";
+import EventEmitter from "eventemitter3";
 
 import { Provider } from "./provider";
+import { CounterfactualEvent } from "./types";
+
+export enum AppInstanceEventType {
```

Comment:
Why not just name it `AppInstanceEvent`?

`AppInstanceEventType.UPDATE_STATE` vs `AppInstanceEvent.UPDATE_STATE`

Reply:
It's inconsistent and ambiguous. Though I too would like a shorter name for it.

Reply:
I may be missing some context.
- Where does the ambiguity arise? The enum is defining events directly relevant to an AppInstance
- And inconsistent with what?

Reply:
Ambiguous - is AppInstanceEvent an enum or the actual event object?
Inconsistent - We have EventType for the type of CounterfactualEvent, but AppInstanceEvent for the type of ... app instance event?